packages/cosmic-swingset/src/sim-params.js

```diff
@@ -32,10 +32,12 @@ export const defaultBeansPerVatCreation =
 // Fees are denominated in units of $1 RUN.
 export const defaultBeansPerFeeUnit = 1_000_000_000_000n; // $1
 
-export const defaultBeansPerInboundTx = defaultBeansPerFeeUnit / 100_000n; // $0.00001
-export const defaultBeansPerMessage = defaultBeansPerFeeUnit / 1_000_000n; // $0.000001
-export const defaultBeansPerMessageByte = defaultBeansPerFeeUnit / 50_000_000n; // $0.0000002
-export const defaultBeansPerMinFeeDebit = defaultBeansPerFeeUnit / 4n; // $0.25
+// TODO: create the cost model we want, and update these to be more principled.
```

Comment:
Where is the relationship between these JS defaults and the matching Go ones in sim-params.go documented? Maybe both should have a pointer to the other, with a note explaining when/why one might take precedence?

Reply:
I've renamed `sim-params.go` to be `default-params.go` since it's used for the chain's default `genesis.json` params.  `sim-params.js` is used for `cosmic-swingset`s `sim-chain` params.

Reply:
(They are disjoint: one's for the chain, the other for the sim.)